pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1002,7 +1024,9 @@ func (n *NetworkPolicyController) processNextAppliedToGroupWorkItem() bool {
 func (n *NetworkPolicyController) syncAddressGroup(key string) error {
 	startTime := time.Now()
 	defer func() {
-		klog.V(2).Infof("Finished syncing AddressGroup %s. (%v)", key, time.Since(startTime))
+		d := time.Since(startTime)
+		metrics.DurationAddressGroupSyncing.Observe(float64(d.Milliseconds()))
```

Comment:
It seems `DurationAddressGroupSyncing` would be not much different from `DurationAddressGroupProcessing` except the latter includes the time to get item from queue, which is not really useful as we don't care how long we wait for upstream events, so I think `DurationAddressGroupProcessing` is not needed.

Reply:
@weiqiangt: what is the status of this PR? Are you still working on it?

Reply:
+1 same applies for AppliedToGroup and InternalNetworkPolicy

Reply:
Thanks, removed `Duration*Processing`.